freqtrade/rpc/rpc_manager.py

```diff
@@ -23,6 +23,12 @@ def __init__(self, freqtrade) -> None:
             from freqtrade.rpc.telegram import Telegram
             self.registered_modules.append(Telegram(freqtrade))
 
+        # Enable Webhook
+        if freqtrade.config.get('webhook', {}).get('enabled', False):
+            logger.info('Enabling rpc.webhook ...')
+            from freqtrade.rpc.webhook import Webhook
```

Comment:
thinking out lout that could we do the imports at the top level, it always feel a bit "iffy" when you have imports inside the classes. Anyway not in the scope of this PR anyway.